include/mptcpd/types.h

```diff
@@ -50,18 +50,39 @@ typedef uint8_t mptcpd_aid_t;
  */
 typedef uint32_t mptcpd_flags_t;
 
-/// Trigger announcement of a new local IP address.
+/**
+ * @brief Trigger announcement of a new local IP address.
+ *
+ * @note Do not use with @c MPTCPD_ADDR_FLAG_FULLMESH.
+ */
 #define MPTCPD_ADDR_FLAG_SIGNAL  (1U << 0)
 
 /// Create a new subflow.
 #define MPTCPD_ADDR_FLAG_SUBFLOW (1U << 1)
 
 /// Set backup priority on the subflow.
 #define MPTCPD_ADDR_FLAG_BACKUP  (1U << 2)
+
+/**
+ * @brief Add remote address to in-kernel fullmesh path management.
```

Comment:
I think the most complete description of the fullmesh flag is in the iproute2 `man ip mptcp` page:

       fullmesh
              If this is a subflow endpoint and additional subflow creation is
              allowed by the MPTCP limits, the MPTCP path manager will try to
              create an additional subflow for each known peer address, using
              this endpoint as the source address. This will occur after the
              MPTCP connection is established. If the peer did not announce
              any additional addresses using the MPTCP ADD_ADDR sub-option,
              this will behave the same as a plain subflow endpoint. When the
              peer does announce addresses, each received ADD_ADDR sub-option
              will trigger creation of an additional subflow to generate a
              full mesh topology.

(see https://git.kernel.org/pub/scm/network/iproute2/iproute2-next.git/tree/man/man8/ip-mptcp.8#n165)

This `@brief` line should say this _local_ address is the one added to in-kernel fullmesh path management.